crates/aptos-dkg/src/pvss/chunked_elgamal_field/chunked_elgamal.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::algebra::homomorphism::{self, FixedBaseMSM};
+use ark_ec::{pairing::Pairing, CurveGroup, VariableBaseMSM};
+
+#[allow(non_snake_case)]
+pub struct Map<'a, E: Pairing> {
+    pub g_1: &'a E::G1Affine,
+    pub h_1: &'a E::G1Affine,
+    pub ek: &'a [E::G1Affine],
+}
+
+type MatrixVectorPair<T> = (Vec<Vec<T>>, Vec<T>);
+type MSMInputVec<'a, E> = MatrixVectorPair<(
+    Vec<<Map<'a, E> as FixedBaseMSM>::Base>,
+    Vec<<Map<'a, E> as FixedBaseMSM>::Scalar>,
+)>;
+
+impl<'a, E: Pairing> Map<'a, E> {
+    pub fn prep_msms(&self, input: &<Self as homomorphism::Map>::Domain) -> MSMInputVec<'a, E> {
+        let first_elgamal_comp = input
```

Comment:
I think you should name `first_elgamal_comp` and `second_elgamal_comp` less verbosely.

e.g., for normal ElGamal, in your PVSS write-up you use `C` to denote the first `m G + r EK` component and `R` to denote the second `r H` component. 

(It's good to maintain consistency between notation here and notation in our algorithms.)